examples/platform/silabs/SiWx917/BUILD.gn

```diff
@@ -14,10 +14,40 @@
 
 import("//build_overrides/chip.gni")
 import("//build_overrides/efr32_sdk.gni")
+import("${chip_root}/examples/common/pigweed/pigweed_rpcs.gni")
 import("${chip_root}/src/lib/lib.gni")
 import("${chip_root}/src/platform/device.gni")
+import("${chip_root}/third_party/silabs/silabs_board.gni")
 import("${efr32_sdk_build_root}/SiWx917_sdk.gni")
+silabs_common_plat_dir = "${chip_root}/examples/platform/silabs"
+wifi_sdk_dir = "${chip_root}/src/platform/silabs/SiWx917/wifi"
+
+declare_args() {
+  enable_heap_monitoring = false
+
+  # OTA timeout in seconds
+  OTA_periodic_query_timeout = 86400
```

Comment:
consider adding the unit of measurement in names. Easier to see what `ota_period_query_timeout_sec` does than `ota_period_query_timeout`. I also believe OTA should be lowercase as all our args seem that way.